docs/howtos/aws_iam_integration.md

```diff
@@ -154,3 +172,28 @@ helm upgrade epinio epinio/epinio \
     --set server.stagingServiceAccountName=epinio-staging-service-account \
     --wait
 ```
+
+Once finished, to remove everyhing remember to remove Epinio, delete service account, detach and delete policies:
```

Comment:
typo in `everything`

Reply:
thanks. fixed in:  2282f595723662907c4cdfe934fd0b367fc57370